### PR TITLE
Add product gridd

### PR DESCRIPTION
**Pull request by dnewh**
This PR adds functionality to the AddProductForm component to submit a Sabre transaction that adds a Grid Product to a circuit.

# Testing
- Navigate to the `/grid` directory
- Run `docker-compose -f examples/splinter/docker-compose.yaml up --build`
- Once all the containers have started follow the Grid on splinter directions until step 6
- In a browser, navigate to http://localhost:3030 and create a new user
- Navigate to the profile page and add a new key pair with the keys generated while setting up Grid
- Navigate to the product sapling and make sure that the service you created shows up in the dropdown at the top.
- Add a product with one of the following valid GTINs `013600000929` or `614141000449`

## Notes

There is a known bug where creating a second service or submitting to multiple services at once causes some data to be lost or overwritten.

### CODE REVIEW

**Comment by dnewh** [comment]
This PR is built on top of PR #224

File: grid-ui/saplings/product/src/components/MultiSelect.scss [hunk 1]
```diff
@@ -33,6 +33,7 @@
       outline: none;
       transition: border-color 0.2s;
```
Comment (eloaverona) [inline grid-ui/saplings/product/src/components/MultiSelect.scss:34]: These changes don't see related to the commit message.

File: grid-ui/saplings/product/src/api/transactions.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  const payloadBytes = protos.ProductPayload.encode({
+    action: protos.ProductPayload.Action.PRODUCT_CREATE,
+    timestamp: Date.now(),
+    productCreate: product
+  }).finish();
+
+  const txn = new SabreTransactionBuilder({
+    name: 'grid_product',
+    version: '1.0',
+    prefix: '621dee'
```
[the remark this answers is not in the document]
Comment (dnewh) [reply grid-ui/saplings/product/src/api/transactions.js]: updated

File: grid-ui/saplings/product/src/api/transactions.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  return `${prefix}00000000000000000000000000000000000000000000${productId}00`;
+}
+
+function buildOrganizationAddess(orgName) {
+  const hash = crypto
+    .createHash('sha512')
+    .update(orgName)
+    .digest('hex')
+    .substring(0, 62);
+  return `cad11d01${hash}`;
```
[the remark this answers is not in the document]
Comment (dnewh) [reply grid-ui/saplings/product/src/api/transactions.js]: updated

**Comment by eloaverona** [comment]
I was able to test and it seems to work! 
Just a couple things:
 - The styling of the form seems a little funky. Like the `Review and Submit` overflowing out of the modal:
Screen Shot 2020-04-16 at 11 31 52 AM

- And if possible, it would be ideal if the form could fit in the modal without having to scroll down. It does not make for a great experience that you can see so little of the form without scrolling down. 
- Also, I suggest that data validation should happen at each step. It feels weird that I have to get to the `Review and Submit` to see warnings about what was wrong in previous steps. In particular because there is no indication of which step the error was. 
Screen Shot 2020-04-16 at 11 35 19 AM

**Pull request merged**